source/tutorials/pd-mice.rst

```diff
@@ -0,0 +1,749 @@
+
+.. command-block::
+
+   qiime longitudinal first-distances \
+     --i-distance-matrix ./core-metrics-results/unweighted_unifrac_distance_matrix.qza \
+     --m-metadata-file ./metadata.tsv \
+     --o-first-distances ./from_first_unifrac.qza \
+     --p-state-column days_post_transplant \
+     --p-individual-id-column mouse_id
+
+We can again use volatility analysis to visualize the change in beta diversity based on distance.
+
+.. command-block::
+
+   qiime longitudinal volatility \
+     --m-metadata-file ./metadata.tsv \
+     --m-metadata-file ./from_first_unifrac.qza \
+     --p-state-column days_post_transplant \
+     --p-individual-id-column mouse_id \
+     --p-default-metric Distance \
+     --o-visualization ./from_first_unifrac_vol.qzv
+
+A linear mixed effects (LME) model lets us test whether there’s a relationship between a dependent variable and one or more independent variables in an experiment using repeated measures. Since we’re interested in genotype, we should use this as an independent predictor.
+
+For our experiment, we’re currently interested in the change in distance from the initial timepoint, so we’ll use this as our outcome variable (given by ``--p-metric``).
+
+``q2-longitudinal`` also requires a state column (``--p-state-column``) which designates the time component in the metadata, and an individual identifier (``--p-individual-id-column``). Which columns should we use in our data?
+
+We can build a model either using the ``--p-formula`` parameter or the ``--p-group-columns`` parameter. For this analysis, we’re interested in whether genotype affects the longitudinal change in the microbial community. However, we also know from our cross sectional analysis that donor plays a large role in shaping the fecal community. So, we should also probably include that in this analysis. We may also want to consider cage effect in our experiment, since this is a common confounder in rodent studies. However, the original experimental design here was clever: although cages were grouped by donor (mice are coprophagic), they were of mixed genotype. This partial randomization helps limit some of the cage effects we might otherwise see.
+
+Based on the experimental design, what group columns should we choose?
+
+.. command-block::
+
+   qiime longitudinal linear-mixed-effects \
+     --m-metadata-file ./metadata.tsv \
+     --m-metadata-file ./from_first_unifrac.qza \
+     --p-metric Distance \
+     --p-state-column days_post_transplant \
+     --p-individual-id-column mouse_id \
+     --p-group-columns genotype,donor \
+     --o-visualization ./from_first_unifrac_lme.qzv
+
+Now, let’s look at the results of the models.
+
+.. question::
+
+   Open the distance volatility plot (``./from_first_unifrac_vol.qzv``) using the qiime 2 viewer. Based on the volatility plot, does one donor change more over time than the other? What about by genotype? Cage?
```

Comment:
```suggestion
   Based on the volatility plot, does one donor change more over time than the other? What about by genotype? Cage?
```

Reply:
Should this be moved up in the tutorial? Like, before we run the LME action?

Reply:
Conceptually, I like combining visualisations with statistical models because I think visualisations can help give you the intuition for the statistics. So, it may be better to have it with the volitility plots int he workshop setting, and may be something to keep in a different order for teaching